lib/types/TLength.ts

```diff
@@ -0,0 +1 @@
+export type TLength = number | string;
```

Comment:
Since this is a specific type, and not a generic type parameter, this shouldn't be named `TLength`, but just `Length` instead.

```suggestion
export type Length = number | string;
```

Besides the type's name, the file also needs to be renamed.